### PR TITLE
adds terraform configs for staging cloudbuild triggers

### PR DESCRIPTION
**Pull request by sjahl**
works towards #78

### CODE REVIEW

**Comment by sjahl** [comment]
Hi all, this config should add our current staging cloudbuild triggers to terraform. To import the current triggers without replacing them, the process is first to use `terraform import` to associate the existing triggers with the terraform objects, and then I'll `terraform apply` once I've done the import and rectified any differences between the configs here and the real resources.

File: terraform/stage/cloudbuild_triggers.tf [hunk 1]
```diff
@@ -33,3 +33,65 @@ resource "google_cloudbuild_trigger" "architecture_helm_lint" {
...
+    push {
+      branch = "^master$"
+    }
+  }
+
+  filename = "cloudbuild.yaml"
+}
+
+# clinvar raw build
+resource "google_cloudbuild_trigger" "clinvar_raw_stage" {
```
Comment (theferrit32) [inline terraform/stage/cloudbuild_triggers.tf:70]: This repository is obsolete and the trigger can be removed. I'll remove it from the project in the console too.

Comment (theferrit32) [reply terraform/stage/cloudbuild_triggers.tf:70]: Not sure what happened to the line selection. This comment is referring to the `clinvar-raw-stage-build` trigger on the `clinvar-scv` repo

**Review by theferrit32 — APPROVED** [review]

**Comment by sjahl** [comment]
These are imported and applied. Made one small fixup commit to correct things that weren't caught with a `terraform plan`

**Pull request merged**